### ISSUE TITLE
Signal 6.6.0 on F37 Desktop does not sync thread after coming out of sleep 

### ISSUE THREAD

**Issue by RichardMyersGV** [flathub/org.signal.Signal#399]
On Flatpak Signal 6.6.0 on Fedora 37, Signal does not sync messages on a thread once the laptop comes out of sleep.  

Signal must be closed, then re-started before the 'new' messages in the thread are seen.  In this case, the new messages included pictures, videos, and test messages, if that makes a difference.  I haven't checked whether it works properly with just text messages which showed up during sleep.

For cases like this, it would be great to have a 'Sync Messages' button to grab it by the short and curlies and make it sync without having to shut down and restart.  But I reckon it should realize on its own that it is out of sync.  Still a button would be nice...

**Comment by bermeitinger-b** [comment]
Are you sure it's an error of the flatpak version? Is there anything in the log? I would please redirect you to the upstream issue tracker.

**Comment by RichardMyersGV** [comment]
I will report it there...  Also, it seems more complicated, as far as what triggers it.  Tried another test with a different thread, and different participants, and it behaved as expected.  Still trying to distinguish exactly what conditions caused this issue to manifest...

**Comment by bermeitinger-b** [comment]
Signal is quite verbose in its logging. You might find something there.

It should be reproducible on the official version, otherwise the upstream people will close your issue right away as unsupported.

**Comment by RichardMyersGV** [comment]
Right, thanks!  I'll try that, and try to isolate what caused the issue before the upstream folks close it as unreproducible.
